pegasus/router.rb

```diff
@@ -33,8 +33,34 @@ def http_vary_add_type(vary,type)
   types.push(type).join(',')
 end
 
+DEFAULT_DOCUMENT_PROXY_MAX_AGE = 1800
```

Comment:
Instead of defining constants here, perhaps you just want to make the defaults based on settings as before?